libafl_nyx/src/settings.rs

```diff
@@ -0,0 +1,73 @@
+const DEFAULT_INPUT_BUFFER_SIZE: u32 = 1024 * 1024;
+const DEFAULT_TIMEOUT_SECS: u8 = 2;
+const DEFAULT_TIMEOUT_MICRO_SECS: u32 = 0;
+
+#[derive(Debug, Clone, Copy)]
+pub struct NyxSettings {
```

Comment:
This should probably use a builder pattern (derived `typed_builder`) instead of having a `new` function, since it has a lot of settings

Reply:
Done, didn't know about `typed_builder`. Very handy, thanks!